src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JDepsGenerator.kt

```diff
@@ -84,4 +80,16 @@ internal class JDepsGenerator @Inject constructor(
             FileOutputStream(Files.createFile(it).toFile()).use(jdepsContent::writeTo)
         }
     }
+}
+
+/**
+ * Extract the normalized major version from the java.version property.
+ *
+ * This is complex as later versions stopped reporting "1.x.y" format in favor of "x.y", so check
+ * for a major version of "1" and if it's so, use the minor version as the major one.
+ */
```

Comment:
can you add a comment here that the assumption here is called on the `java.version` property, and is assumed to be a well-formed version number?

Reply:
Sure!

Reply:
Wait, I'm confused - isn't that what this comment says, if taken with the previous sentence (which refers to the java.version property directly)?

Reply:
I reworded the text. Hopefully that makes more sense.